psychopy/hardware/keyboard.py

```diff
@@ -83,6 +83,15 @@
 
 defaultBufferSize = 10000
 
+# monkey-patch bug in PTB keyboard where winHandle=0 is documented but crashes
+if havePTB and sys.platform=='win':
```

Comment:
Typo in sys.platform check, change to:

`if havePTB and sys.platform=='win32':`

and it works great.

Reply:
doh! thanks